package.json

```diff
@@ -21,6 +21,7 @@
     "eslint-fix": "npm run eslint -- --fix",
     "prettier": "prettier --check .",
     "prettier-write": "prettier --write .",
+    "test-only": "jest --watch",
```

Comment:
Only `jest` in `test-only`.
We can have another script with [`--watchAll`](https://jestjs.io/docs/cli#--watchall)